### PR TITLE
Remove flake8

### PR DESCRIPTION
**Pull request by kosmos342**
All requirement.txt is inserted to `setup.install_requires`
https://github.com/yola/yodeploy/blob/85a5d249c3dee50386c249e2adf9495cef362e79/setup.py#L8-L18

There are some problems with installing `flake8==3.0.4`
```
ERROR:yodeploy.virtualenv:flake8 3.0.4 requires configparser, which is not installed.
```
Tried locally with python2 env
`pip install -i https://<user>:<password>@<yolapi host>/simple/ flake8==3.0.4`
It throw same error. Upgrading pip and setuptools doesn't help

anyway we have `snitch` for that

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,6 @@ Flask-HTTPAuth==2.5.0
 Tempita==0.5dev
 boto==2.42.0
 coverage==4.2.0
-flake8==3.0.4
 gunicorn==19.9.0
 mock==1.0.1
```
Comment (stefanor) [inline requirements.txt]: Really, this shouldn't be part of `install_requires` either...

Comment (kosmos342) [reply requirements.txt]: I'll try to split requirements in another PR

Comment (kosmos342) [reply requirements.txt]: I plan to split deps to base_requirements.txt and test_requirements.txt. set base_requirement in setup.require and test_requirements in setup.test_require. and requirements.txt include both files

Comment (RayeN) [reply requirements.txt]: > I plan to split deps to base_requirements.txt and test_requirements.txt.

Why do we need that?  I would follow our usual procedure.

Comment (kosmos342) [reply requirements.txt]: set all requirements(exclude test requirements) in setup.pu?

Comment (RayeN) [reply requirements.txt]: Yup. That's what we usually do.  It allows us to use specific versions for testing and a range to be installed along with the lib.

Comment (kosmos342) [reply requirements.txt]: What is the process to build virtualenv. As I see firstly we install yodeploy and then in separate command we install <service>/requirements.txt. Is it right? In this case setting range for requirements doesn't do effect.

Comment (RayeN) [reply requirements.txt]: It's probably the same as with any other lib, nope?

**Pull request merged**